src/main.tsx

```diff
@@ -5,6 +5,7 @@ import "primereact/resources/primereact.min.css";
 import "primereact/resources/themes/lara-light-indigo/theme.css";
 import DcOpenOrders from "./components/DcOpenOrders.tsx";
 import BarChart from "./components/charts/BarChart.tsx";
+import "primereact/resources/themes/saga-blue/theme.css";
```

Comment:
why are u adding this theme ?

Reply:
I added this when primereact styles were not reflecting in my code. Will remove this.